cmake/HPX_AddConfigTest.cmake

```diff
@@ -74,6 +74,7 @@ macro(add_hpx_config_test variable)
 
       set(_msg "${_msg} - ${_run_msg}")
     else()
+      message("... ${${variable}_OUTPUT}")
       set(_msg "${_msg} - Failed")
```

Comment:
You are right of course. This can be completely removed as it is redundant anyway. Somehow crawled into the changeset.